src/App.tsx

```diff
@@ -23,6 +24,12 @@ function App() {
     setIsLogedin(accessToken ? true : false);
   }, [accessToken]);
 
+  useEffect(() => {
+    if (state.isLogedin && accessToken) {
```

Comment:
If I am correct on the flow, We don't need to check the auth here, this page only be accessible if user is logged in .